e2e/e2e_v113_test.go

```diff
@@ -30,7 +30,8 @@ func testPionE2ESimpleED25519(t *testing.T, server, client func(*comm)) {
 		dtls.TLS_ECDHE_ECDSA_WITH_AES_128_CCM_8,
 		dtls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
 		dtls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
-		dtls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
+		// Is TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 allowed with ed25519 keys?
```

Comment:
Eh... yeah. I don't think it should be. The server key needs to be an RSA key in that case, which is no true here.

Reply:
@lukaslihotzki Looking really good! Thanks for fixing up so quick.